modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -293,15 +293,15 @@ GrainTracker::finalize()
 
   if (_ebsd_reader && _first_time)
   {
-    expandEBSDGrains();
+    expandPointHalos();
 
     /**
      * By expanding the EBSD Grains we've effectively erased one level of halo.
```

Comment:
That's the goal, this PR won't get us there yet. I plan to modify the existing EBSD IC (ReconVarIC) and eventually remove all traces of this logic from the GrainTracker!